### DISCUSSION TITLE
PyAnatomist all the steps to run my own .py script using anatomist.api?

### DISCUSSION THREAD

**Discussion by ece7048** [brainvisa/casa-distro#323] [Q&A] [answered]
Dear Brainvisa community,

I want to install the development singularity image of brainvisa and run one python script to use anatomist API and soma API.
I am trying to do this in a university cluster with ssh.
The university cluster OS is:
_NAME="Scientific Linux"
VERSION="7.9 (Nitrogen)"
ID="scientific"
ID_LIKE="rhel centos fedora"
VERSION_ID="7.7"
PRETTY_NAME="Scientific Linux 7.9 (Nitrogen)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:scientificlinux:scientificlinux:7.9:GA"
HOME_URL="http://www.scientificlinux.org//"
BUG_REPORT_URL="mailto:scientific-linux-devel@listserv.fnal.gov"

REDHAT_BUGZILLA_PRODUCT="Scientific Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.9
REDHAT_SUPPORT_PRODUCT="Scientific Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.9"_

I am using python==3.8.3
and I am trying to install in an anaconda3 environment.

Till now I did the follow steps:
1) download the casa-dev-5.0-11.sif and casa-dev-5.0-11.sif.json
2) singularity run -c -B casa_distro/brainvisa-opensource-master:/casa/setup ./casa_distro/casa-dev-5.0-11.sif branch=master distro=bvdev
3) export PATH="$HOME/casa_distro/brainvisa-opensource-master/bin:$PATH"
4) bv_maker (p.s. the bv_maker doc is not implemented correctly but the 3 previous step was ok!!)
5) pip install pyqt5

All these steps are successfully implemented.

When I am trying to run:
1) python3 
2)>> import anatomist.api as anatomist

**Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'anatomist'**

if i try:
1) ipython --gui=qt
2) import anatomist.api as anatomist

**ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-3b693a10658e> in <module>
----> 1 import anatomist.api as anatomist

ModuleNotFoundError: No module named 'anatomist'**

Any idea why there is no bidding with the PyAnatomist API??

Thank you a lot!

**Comment by denisri** [comment, marked as answer]
duplicate of https://github.com/brainvisa/brainvisa.github.io/discussions/121, let's continue there.
